packages/core/test/stateTree.test.ts

```diff
@@ -17,21 +17,29 @@ describe('State tree', function () {
     this.timeout(30 * 60 * 1000)
 
     let unirepContract
-    let stateTree
+    let config
```

Comment:
This doesn't need to be a global variable.

Reply:
[5f4ff50](https://github.com/Unirep/Unirep/pull/243/commits/5f4ff505d0f5fc341c1f75c9e8cdcab1d6c82606)